keps/sig-api-machinery/4355-coordinated-leader-election/kep.yaml

```diff
@@ -0,0 +1,34 @@
+title: Coordinated Leader Election
+kep-number: 4355
+authors:
+  - "@jpbetz"
+owning-sig: sig-api-machinery
+participating-sigs:
```

Comment:
Should we include sig-cluster-lifecycle as participating sigs?

Reply:
Added. Thanks!